prov/rxm/src/rxm_fabric.c

```diff
@@ -73,6 +82,48 @@ static struct fi_ops rxm_fabric_fi_ops = {
 	.ops_open = fi_no_ops_open,
 };
 
+static int rxm_fabric_init_util_coll(struct rxm_fabric *fabric)
+{
+	struct fi_info *hints, *coll_info;
+	struct fid_fabric *coll_fabric;
+	int ret;
+
+	hints = fi_allocinfo();
+	if (!hints)
+		return -FI_ENOMEM;
+
+	hints->fabric_attr->prov_name = strdup(OFI_OFFLOAD_PREFIX "coll");
```

Comment:
coverity is probably going to complain that we didn't check for failure

Reply:
Adding a check is easy.